pyproject.toml

```diff
@@ -45,6 +45,8 @@ fsspec = { version = "2022.8.2" }
 
 trino = { version = "0.319.0", optional = true, extras = ['sqlalchemy'] }
 
+snowflake-connector-python = { version = "^3.4.0" }
```

Comment:
you need to declare this as optional and add to the appropriate extras section. Remember to run `poetry update` after you apply the change